src/Sulu/Bundle/WebsiteBundle/Resources/public/js/services/reference-store.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(function() {
+
+    'use strict';
+
+    var synonyms = {},
+        references = {},
+
+        resolveAlias = function(alias) {
```

Comment:
I know that we spoke about it, but I still don't like the idea 😄 Just an idea (open to discuss): Would it work to add both `articles` and `article` tags to the service definition in the article bundle? Would maybe work both ways then, and both "different" services would use the same instance. Then we wouldn't have to introduce something like this to the core.